src/System.Private.Xml/src/System/Xml/Serialization/XmlSerializer.cs

```diff
@@ -1040,16 +1040,16 @@ public static string GetXmlSerializerAssemblyName(Type type, string defaultNames
 
         /// <include file='doc\XmlSerializer.uex' path='docs/doc[@for="XmlSerializer.CreateReader"]/*' />
         ///<internalonly/>
-        protected virtual XmlSerializationReader CreateReader() { throw new PlatformNotSupportedException(); }
+        protected virtual XmlSerializationReader CreateReader() { throw new NotImplementedException(); }
```

Comment:
Yes, the method must always be overridden. It's an extension point of XmlSerializer. And this is what we have on Desktop.